Makefile

```diff
@@ -189,11 +189,15 @@ $(BINDIR)/e2e.test: .init
 	# generate all pkg/client contents
 	$(DOCKER_CMD) $(BUILD_DIR)/update-client-gen.sh
 	# generate openapi
-	# $(DOCKER_CMD) $(BINDIR)/openapi-gen \
-	# 	--v 1 --logtostderr \
-	# 	--go-header-file "vendor/github.com/kubernetes/repo-infra/verify/boilerplate/boilerplate.go.txt" \
-	# 	--input-dirs "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1,k8s.io/client-go/pkg/api/v1,k8s.io/apimachinery/pkg/apis/meta/v1" \
-	# 	--output-package "github.com/kubernetes-incubator/service-catalog/pkg/openapi"
+	$(DOCKER_CMD) $(BINDIR)/openapi-gen \
+		--v 1 --logtostderr \
+		--go-header-file "vendor/github.com/kubernetes/repo-infra/verify/boilerplate/boilerplate.go.txt" \
+		--input-dirs "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1" \
+		--input-dirs "github.com/kubernetes-incubator/service-catalog/pkg/apis/settings/v1alpha1" \
+		--input-dirs "k8s.io/client-go/pkg/api/v1" \
+		--input-dirs "k8s.io/client-go/pkg/api" \
+		--input-dirs "k8s.io/apimachinery/pkg/apis/meta/v1" \
+		--output-package "github.com/kubernetes-incubator/service-catalog/pkg/openapi"
```

Comment:
does this require to be moved up here? I don't think there are any dependencies in either direction, but I'd rather not move things around if they don't need it.